addons/website_slides_survey/controllers/slides.py

```diff
@@ -74,13 +75,9 @@ def create_slide(self, *args, **post):
         # Then create the slide
         result = super(WebsiteSlidesSurvey, self).create_slide(*args, **post)
 
-        if create_new_survey:
-            # Set the redirect_url used in toaster
-            action_id = request.env.ref('survey.action_survey_form').id
-            result.update({
-                'redirect_url': '/web#id=%s&action=%s&model=survey.survey&view_type=form' % (post['survey_id'], action_id),
-                'redirect_to_certification': True
-            })
+        if post['slide_category'] == "certification":
+            # Set the url to redirect the user to the survey
+            result['url'] = '/slides/slide/%s?fullscreen=1' % (slug(result['category_id'].browse(result['slide_id']))),
```

Comment:
Browsing from `result['category_id']` seems strange, we should just use request.env['slide.slide']